loader.js

```diff
@@ -18,7 +18,14 @@ nconf.argv().env().file({
 });
 
 var	pidFilePath = path.join(__dirname, 'pidfile');
-var outputLogFilePath = path.join(__dirname, 'logs/output.log');
+
+var outputLogFilePath = path.join(__dirname, nconf.get('logFile') || 'logs/output.log');
+
+var logDir = path.dirname(outputLogFilePath);
+if (!fs.existsSync(logDir)) {
+	fs.mkdirSync(path.dirname(outputLogFilePath), { recursive: true });
```

Comment:
It doesn't appear that the options object is supported on Node 8: https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_fs_mkdirsync_path_mode

So it would be best to just use `mkdirp.sync`